server/views/pages/view/calculationSummary.njk

```diff
@@ -40,17 +42,47 @@
     {% include "../partials/ersedCannotHappenNotificationBanner.njk" %}
     <div class="govuk-grid-row">
         <div class="govuk-grid-column-two-thirds">
-            <h1 class="govuk-heading-xl">Dates {% if model.prisonerDetail %}for {{ model.prisonerDetail.firstName | title + ' ' + model.prisonerDetail.lastName | title }}
-                {% endif %}
-            </h1>
```

Comment:
Guessing this was agreed to remove - it's a bit tricky as the design has moved on with the mini profile etc since

Reply:
Yes, Jo/Ash have been through and updated bits of the design